src/components/MoneyRequestSkeletonView.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import SkeletonViewContentLoader from 'react-content-loader/native';
 import {Rect} from 'react-native-svg';
 import styles from '@styles/styles';
 import themeColors from '@styles/themes/default';
 import variables from '@styles/variables';
+import SkeletonViewContentLoader from './SkeletonViewContentLoader';
```

Comment:
Subpath will be import directly instead of using alias to avoid linting issue
<img width="1152" alt="Screen Shot 2023-10-29 at 15 33 06" src="https://github.com/Expensify/App/assets/16502320/c92219a5-3766-4658-b59c-ebf67d981319">

Reply:
yep, it's expected for now

<img width="873" alt="relative path" src="https://github.com/Expensify/App/assets/108292595/b4b46a71-1328-42be-ae1d-c6b9ce07ca87">